apps/yapms/src/routes/app/+page.svelte

```diff
@@ -0,0 +1,15 @@
+<script lang="ts">
+  import { page } from "$app/stores";
+  const mapID = $page.url.searchParams.get("map");
```

Comment:
My understanding here is that under most circumstances this searchParam wouldn't exist and instead the param would be "m". Perhaps something that got changed halfway through?

Reply:
Yeah, this was just me messing around with something else. I've deleted it now, for this PR.